### PR TITLE
Refine .travis.yml

### PR DESCRIPTION
**Pull request by ishkawa**
- Fix broken build log due to too long log by `pod repo update`.
- Use `pod lib lint` to lint podspec.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,11 +9,13 @@ env:
 before_install:
   - brew update 
   - brew install carthage
+
+install:
   - carthage bootstrap --use-submodules --no-build
+  - pod repo update --silent
```
Comment (ikesyo) [inline .travis.yml]: Is this command still needed? (since `pod lib lint` for the local spec is now used)

**Comment by ikesyo** [comment]
I'm sorry I missed the PR. :bow:

File: .travis.yml (hunk 1 above)
Comment (ishkawa) [inline .travis.yml]: Yes, this is still necessary because lint fails if repo doesn't have specified podspec of Result.

Comment (ikesyo) [inline .travis.yml]: Ah, it makes sense!

**Comment by ikesyo** [comment]
:+1:

**Pull request merged**